resources/migrations/20200427092452-audit-log.down.sql

```diff
@@ -0,0 +1 @@
+DROP TABLE audit_log CASCADE;
```

Comment:
I wonder if we should just stop writing these (potentially data lossing) down migrations especially for high value data.

Reply:
That's a good question. However we use migratus reset to empty the db in some tests, so perhaps a middle ground would be to rename the table here?

Reply:
I guess for tests it may make sense to have some other reset function that is faster i.e. truly start from an empty DB or db dump. I think this can be left here for now.